src/runtime_src/hip/api/hip_stream.cpp

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023-2024 Advanced Micro Device, Inc. All rights reserved.
+
+#include "core/common/error.h"
+
+#include "hip/config.h"
+#include "hip/hip_runtime_api.h"
+
+#include "hip/core/stream.h"
+
+namespace xrt::core::hip {
+static void
+hipStreamCreateWithFlags(hipStream_t *stream, unsigned int flags)
```

Comment:
Suggest `type* var` over `type *var`, which has been done in other place (but maybe not in all :-) )
```suggestion
hipStreamCreateWithFlags(hipStream_t* stream, unsigned int flags)
```